server/src/test/java/org/opensearch/index/engine/InternalEngineTests.java

```diff
@@ -6637,6 +6637,7 @@ public void testLastRefreshCheckpoint() throws Exception {
                 while (done.get() == false) {
                     long checkPointBeforeRefresh = engine.getProcessedLocalCheckpoint();
                     engine.refresh("test", randomFrom(Engine.SearcherScope.values()), true);
+                    assertThat(engine.currentOngoingRefreshCheckpoint(), greaterThanOrEqualTo(engine.lastRefreshedCheckpoint()));
```

Comment:
@ankitkala this assertion seems to be flaky (https://github.com/opensearch-project/OpenSearch/issues/9029), could you please take a look? thank you